### PR TITLE
Merge kickstart config for Azure

### PR DESCRIPTION
**Pull request by szarkos**

### CODE REVIEW

**Comment by szarkos** [comment]
@kbsingh 

This ks is for CentOS 7.  I built the image with virt-install, converted to VHD and successfully tested on Azure.

File: cloudimg/CentOS-7-x86_64-Azure.ks [hunk 1]
```diff
@@ -1,78 +1,90 @@
...
-TYPE="Ethernet"
-USERCTL="yes"
-PEERDNS="yes"
-IPV6INIT="no"
-PERSISTENT_DHCLIENT="1"
-EOF
+%post --erroronfail --log=/var/log/anaconda/post-install.log
+#!/bin/bash
 
-echo "virtual-guest" > /etc/tuned/active_profile
```
Comment (kbsingh) [inline cloudimg/CentOS-7-x86_64-Azure.ks]: are we sure we want to disable this ?

File: cloudimg/CentOS-7-x86_64-Azure.ks [hunk 2]
```diff
@@ -82,73 +94,81 @@ UPDATEDEFAULT=yes
...
 
-yum clean all
+cat << EOF > /etc/sysconfig/network
+NETWORKING=yes
+NOZEROCONF=yes
+HOSTNAME=localhost.localdomain
+EOF
 
-# XXX instance type markers - MUST match CentOS Infra expectation
-echo 'azure' > /etc/yum/vars/infra
```
Comment (kbsingh) [inline cloudimg/CentOS-7-x86_64-Azure.ks]: we should leave this in, it allows the mirrorlist redirector to send the content download urls to the best place

Comment (kbsingh) [reply cloudimg/CentOS-7-x86_64-Azure.ks]: nevermind, i see it being set later.

File: cloudimg/CentOS-7-x86_64-Azure.ks (hunk 1 above)
Comment (kbsingh) [reply cloudimg/CentOS-7-x86_64-Azure.ks]: nevermind, i see it being set later.

**Pull request merged**